viral/basic_plots.py

```
@@ -0,0 +1,938 @@
+            axes[idx].text(i, text_y, p_text, ha="center", va="top")
+
+    plt.tight_layout(rect=[0, 0, 1, 0.95])
+    plt.savefig(
+        SERVER_PATH
+        / "viral_plots"
+        / "firing_rates"
+        / f"comparison_firing_rates_rewarded_{rewarded}.pdf",
+        bbox_inches="tight",
+        transparent=True,
+    )
+    1 / 0
```

Ruff (0.14.14)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)